HACKING.md

```diff
@@ -1,5 +1,127 @@
-# Hacking on the `cluster-logging-operator`
+# Hacking on the Cluster Logging Operator
+
+## Preparation
+
+Clone this repository into `$GOPATH/github.com/openshift/cluster-logging-operator`
+
+Install `podman`. For example on  Fedora:
+```
+sudo dnf install -y podman
+```
+
+Check out some other required repositories in your $GOPATH:
+```
+go get -d -u github.com/openshift/elasticsearch-operator/... github.com/openshift/origin-aggregated-logging/...
+```
+
+**Note:** `podman` copies our source tree very slowly. You can use `docker`
+instead, this will not exactly reproduce how images are built for release or by
+CI, but is close enough for many purposes. Put a symlink early in your path:
+```
+ln -s /usr/bin/podman /somewhere/early/in/my/PATH`
```

Comment:
shouldn't this symlink be targeting the docker binary?  and include podman in the linkname?

Reply:
Yes, good catch.